Source/setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+with open('../README.md', 'r') as rm:
+    long_description = rm.read()
+
+setuptools.setup(
+    name="PyBoy",
+    version="0.0.1",
+    author="Mads Ynddal",
+    author_email="mynddal@me.com",
+    long_description=long_description,
+    content_type="text/markdown",
+    url="https://github.com/Baekalfen/PyBoy",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: Implementation :: PyPy",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 2",
+        "License :: Freely Distributable",
```

Comment:
Creative Commons recommends [using a dedicated software license instead](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software).